test/common/testlib.py

```diff
@@ -534,7 +534,7 @@ def tearDown(self):
         shutil.rmtree(self.tmpdir)
 
     def login_and_go(self, path=None, user=None, host=None, authorized=True):
-        self.machine.start_cockpit(host)
```

Comment:
this is passed to https://github.com/cockpit-project/cockpit/blob/919f083e40b3000549a62d08fcbd7d3aed2a4e5e/test/common/testvm.py#L570 in our tests, I don't think we should remove it at this point